src/components/layout/addons/AddonsLayout.js

```diff
@@ -0,0 +1,266 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import pluralize from 'pluralize';
+import { Input, TableOfContents, global, styles, TagList, TagLink } from '@storybook/design-system';
+import { graphql } from 'gatsby';
+import GatsbyLinkWrapper from '../../basics/GatsbyLinkWrapper';
+import { AddonsLearn } from './AddonsLearn';
+import { AddonsList } from './AddonsList';
+import { AddonsAside, AddonsAsideTitle } from './AddonsAside';
+
+const { breakpoint, spacing, color, pageMargins, typography } = styles;
+const { GlobalStyle } = global;
+
+const Content = styled.main`
+  flex: 1;
+  min-width: 0; /* do not remove  https://weblog.west-wind.com/posts/2016/feb/15/flexbox-containers-pre-tags-and-managing-overflow */
+  max-width: 800px;
+  margin: 0px auto;
+`;
+
+const Wrapper = styled.div`
+  ${pageMargins}
+  padding-bottom: 3rem;
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    display: ${(props) => (props.searchLayout ? 'block' : 'flex')};
+  }
+`;
+
+const Divider = styled.div`
+  border-top: 1px solid ${color.border};
+  margin-top: 24px;
+  margin-bottom: 24px;
+`;
+
+const Sidebar = styled.div`
+  flex: 1;
+  margin: 1rem 0 2rem;
+  padding-bottom: 1rem;
+  border-bottom: 1px solid ${color.border};
+  display: ${(props) => (props.hideSidebar ? 'none' : 'block')};
+  margin-bottom: 32px;
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    flex: 0 0 240px;
+    margin-top: 0;
+    margin-top: 0;
+    margin-left: auto;
+    margin-right: ${(props) => (props.searchLayout ? 'auto' : '20px')};
+    margin-bottom: ${(props) => (props.searchLayout ? spacing.padding.medium : 0)}px;
+    padding-bottom: 0;
+    padding-right: ${(props) => (props.searchLayout ? 0 : 20)}px;
+    border-bottom: none;
+  }
+`;
+
+Sidebar.propTypes = {
+  hideSidebar: PropTypes.bool,
+};
+
+const ToCContent = styled.div`
+  /* Hide ToC on mobile, the primary navigation is search */
+  display: none;
+
+  ${(props) =>
+    props.hideToC
+      ? css`
```

Comment:
How are you deciding when to use `css` in these? This feels a bit out of place as I don't see it elsewhere.

Reply:
I was doing this blindly but, it seems like you only need to do this when interpolating a function. I'll push an update.

Reply:
Np sounds good. I know in previous iterations of the design system that Dom was consistently using css like this but I have noticed it seems to be the exception that you actually need it. Just trying to find common patterns for us.